packages/launchpad/src/migration/OptOutModal.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <StandardModal
+    :model-value="true"
+    :title="t('migration.renameAuto.modal.title')"
+    @update:modelValue="emit('cancel')"
+  >
+    <Alert
+      status="warning"
+      :title="t('migration.renameAuto.modal.warning')"
+      :icon="WarningIcon"
+      icon-classes="icon-dark-orange-400"
+    />
+    <MigrationTitle
+      :title="t('migration.renameAuto.title')"
+      class="mt-24px"
+    />
+    <MigrationList>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line1') }}
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line2') }}
+        <CodeTag class="text-jade-500">
+          [filename].cy.[ext]
+        </CodeTag>
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line3') }}
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line4') }}
+        <CodeTag class="text-jade-500">
+          [filename].cy.[ext]
+        </CodeTag>
+      </MigrationListItem>
+      <MigrationListItem>
+        <i18n-t
+          scope="global"
+          keypath="migration.renameAuto.modal.line5"
+        >
+          <CodeTag class="text-jade-500">
+            [filename].cy.[ext]
+          </CodeTag>
+        </i18n-t>
+      </MigrationListItem>
+      <MigrationListItem>
+        <i18n-t
+          scope="global"
+          keypath="migration.renameAuto.modal.line6"
+        >
+          <CodeTag class="text-jade-500">
+            [filename].cy.[ext]
+          </CodeTag>
+        </i18n-t>
+      </MigrationListItem>
+    </MigrationList>
+    <Radio
+      v-model:value="value"
+      :options="options"
+      name="skipRename"
+      :label="t('migration.renameAuto.modal.label')"
+    >
+      <template #option="{ option }">
+        <span
+          v-if="option.value === 'rename'"
+          class="text-jade-500"
+        >
+          <i18n-t
+            scope="global"
+            keypath="migration.renameAuto.modal.option1"
+          >
+            <CodeTag class="ml-0 text-jade-500">
+              [filename].cy.[ext]
+            </CodeTag>
+          </i18n-t>
+        </span>
+        <span
+          v-else
+          class="text-gray-800"
+        >
+          {{ props.hasCustomIntegrationFolder ? t('migration.renameAuto.modal.option2') : t('migration.renameAuto.modal.option3') }}
+          <i18n-t
+            v-if="value !== 'rename'"
```

Comment:
Nit, but it'd be cool if we updated Radio to provide a `isChecked` value for the option slot. That's really what we're wanting to assert here, that this option is currently selected. What you have works fine, it's just a little indirect.

Reply:
Added 'checked' to the Radio component.